modyn/config/schema/pipeline/sampling/config.py

```diff
@@ -95,6 +95,14 @@ class NewDataStrategyConfig(_BaseSelectionStrategy):
 class CoresetStrategyConfig(_BaseSelectionStrategy):
     name: Literal["CoresetStrategy"] = Field("CoresetStrategy")
 
+    warmup_triggers: int = Field(
+        description=(
+            "Defines for how many epochs no pre- or downsampling is performed. "
```

Comment:
Is this `how many triggers`? i.e. it controls on the trigger level not epoch level

Reply:
yes! thanks for catching this